crates/op-rbuilder/src/monitoring.rs

```diff
@@ -88,10 +86,8 @@ impl Monitoring {
         let num_reverted_tx = decode_chain_into_reverted_txs(chain);
         self.metrics.inc_num_reverted_tx(num_reverted_tx);
 
-        self.execution_outcome
-            .extend(chain.execution_outcome().clone());
         let builder_balance =
-            decode_state_into_builder_balance(&self.execution_outcome, self.builder_signer)
+            decode_state_into_builder_balance(chain.execution_outcome(), self.builder_signer)
```

Comment:
if the builder balance doesn't change in the last block (like it missed a block or something) I don't think the balance will be in the execution outcome from the chain